Added pages for "Contact" and "About"

toby63 opened the pull request:
-Also moves "Sponsors" to the footer menu

The website is missing some general content most websites offer.
It also makes you a bit more trust-worthy and transparent.

Some of it might even be a legal requirement.

Fixes #126 and #98.

Krzmbrzl commented on hugo/config/_default/menus.toml:
```diff
@@ -34,6 +34,18 @@
+    identifier = "about"
+    url = "/about"
+
+[[main]]
+    title = "Contact"
+    weight = 8
+    identifier = "contact"
+    url = "/contact"
```
Imo that should go into the footer instead of the main menu

toby63 replied:
On many websites it is listed in the Menu and honestly I think no is really paying attention to footers (thats probably why so many companies put their legal stuff into that :stuck_out_tongue: ).
But it's up to you.

Krzmbrzl replied:
We simply don't have the space in the menu. Plus most pages I have seen have the contact information in the footer :shrug: 
So I still think that's where we should put it.

toby63 replied:
I would like to have it in the Menu, but having it in the footer is better than not having it at all.

Krzmbrzl reviewed (commented):
The contact details are also listed on the `Contribute` page so if we introduce a special `Contact` page, we should remove the info from there and link to the contact page instead.

For the `About` page I would say that this kind of information is partially covered by the main page. It also contains some words about the project...

toby63 commented:
> The contact details are also listed on the Contribute page so if we introduce a special Contact page, we should remove the info from there and link to the contact page instead.

I disagree, the Contribute page lists these things for community participation.
So the purpose is different and should therefore be mentioned in both cases.

> For the About page I would say that this kind of information is partially covered by the main page. It also contains some words about the project...

As I already said in #98, it does not cover what this should really be about. 
For a software website it is obvious what the software is, but the interesting parts for an about page are: who are you, who is running this etc.

Krzmbrzl commented:
> I disagree, the Contribute page lists these things for community participation.
So the purpose is different and should therefore be mentioned in both cases.

But the means of contact are the same. Thus you'd only be duplicating the information which is never a good idea.

> For a software website it is obvious what the software is, but the interesting parts for an about page are: who are you, who is running this etc.

That's why I wrote _partially_ :point_up:

toby63 commented:
> But the means of contact are the same. Thus you'd only be duplicating the information which is never a good idea.

No.
You have:
1. Community: Which means users (and team)
2. Contact: Which means searching for contact to the team.

> That's why I wrote partially :point_up:

I saw that, but it seemed, that you used it as an argument against an about page.
So thats the point.

Krzmbrzl commented:
> No.
You have:
    Community: Which means users (and team)
    Contact: Which means searching for contact to the team.

There no reason why these two ways of contacting can't live on the same page.

> It is done very often.

It is indeed. But it is never good to do that.

> Of course information can be outdated here and be new there, but your website is quite small, so it should be manageable.

But it has to be done and especially: remembered. That's the issue

toby63 commented:
> There no reason why these two ways of contacting can't live on the same page.

There is, those are two different categories:
1. Contribution -> Community (Channels)
2. Contact to the Mumble Team

I don't see why and how that can be on the same page, also because there are different contents for both.

Krzmbrzl commented:
They don't differ as we as a team are also reachable on the e.g. IRC. Thus effectively there is no difference. Any user can choose what they think is an appropriate means of communicating with us.

toby63 commented:
> They don't differ as we as a team are also reachable on the e.g. IRC. Thus effectively there is no difference. Any user can choose what they think is an appropriate means of communicating with us.

Well you mean a different level than I do, but instead of continuing this.

What would be your solution?
Should I link to the Community section (in Contribute) on the contact page?
Or vice versa?
Or what is your exact idea?

Krzmbrzl commented:
> Or what is your exact idea?

My idea is to summarize all means of contacting in the `Contact` page and then linking to that every time we want to provide contact information (e.g. from the `Contribute` page).

> Another idea would be some kind of template that could be included, like the download templates.
Of course it has to be a bit different, just like a text box.
This way you would have the list in one place.

Definitely a lot better than duplicating the information in source. However I think it is better to link to a contact page instead. I mean that's kinda what links are for :shrug:

toby63 commented:
> My idea is to summarize all means of contacting in the Contact page and then linking to that every time we want to provide contact information (e.g. from the Contribute page).

If thats what you want; I will at least add a seperate category to the Contact page to seperate between `Community Channels` and other Contact possibilities like Email etc.

Krzmbrzl commented:
> If thats what you want; I will at least add a seperate category to the Contact page to seperate between Community Channels and other Contact possibilities like Email etc.

Sounds good.

> What about the About page now?

Well idk what I would want to put there since I don't think this to be vital in the first place. But then again I never look at About pages of other projects (as long as I find information on the actual software I am interested in elsewhere) :shrug:

Krzmbrzl commented on hugo/content/about.md:
```diff
@@ -0,0 +1,73 @@
+
+You find the Source Code and further instructions regarding Building and Development in our [GitHub Repo](https://github.com/mumble-voip).
+
+## Licenses
+
+<!-- https://github.com/mumble-voip/mumble/blob/master/LICENSE -->
+
+<!-- more licenses? should be as correct and complete as possible -->
```
I think we don't need to specify licenses here explicitly. If you think it's important you can state something like "We are generally using Open Source license without copy-left clauses (e.g. BSD-3)" but in the end the user can simply checkout the license for the individual projects.

toby63 replied:
Then I will link to Github and tell people to look at the licenses of each project.
Ok?

Krzmbrzl replied:
Sounds good

Krzmbrzl commented on hugo/content/about.md:
```diff
@@ -0,0 +1,73 @@
+
+We are a team of voluntary developers - both professional and non-professional coders - from different countries - including Germany, Italy, USA and more.
+
+<!--
+Link to Github Team page:
+https://github.com/orgs/mumble-voip/people
+
+-->
```
Probably a good idea :+1: 

The list is effectively outdated though (in terms of who is still active) since the majority of people listed there have not contributed to the project since I started working on it.

toby63 replied:
> The list is effectively outdated though

Thats a problem.
If we list the Core team, the information should be fairly recent imo.

Krzmbrzl replied:
I would simply not list that. I think the contributors page on GitHub contains all information that a user might want to know. Thus imo it should be enough to link to that :shrug:

toby63 replied:
I disagree:
1. The Github Graph shows some inactive team members first.
2. One of the main intentions of an about page is to inform users, who the people behind the project are and this would be done best by listing the core team.

But if you don't want that, there is of course nothing I can do :rabbit:.

Krzmbrzl replied:
Fair enough. In that case I think we could list
- @davidebeatrici - Core developer
- @Krzmbrzl - Core developer, GitHub Management
- @ZeroAbility - Release Management
- @Kissaki  - Community work
- @Natenom - Testing

I think as far as the core team and their current main tasks are concerned, that should be it (correct me if that's not the case).

toby63 replied:
I assume you want no real names (though they can be found out easily anyway) ?
I will link to the Github profiles then.

> correct me if that's not the case

I have no clue :shrug:. But doesn't Kissaki & ZeroAbility do other things as well?

Krzmbrzl replied:
If the names are available on the respective GitHub profile pages (which should be linked in any case), I think it is fine to also list the real names on the Website. If the real name is only contained in the commits, I would say that they should not be mentioned on the website.

> I have no clue shrug. But doesn't Kissaki & ZeroAbility do other things as well?

This comment was actually more directed towards the mentioned team members. I didn't expect you to know that :grin:

toby63 replied:
> If the names are available on the respective GitHub profile pages (which should be linked in any case), I think it is fine to also list the real names on the Website

Ok, sounds good. Real names are more trustworthy.

> This comment was actually more directed towards the mentioned team members. I didn't expect you to know that

I already assumed that. But because they do respond very rarely (which is to some degree understandable if you got much to do and get many messages), it is probably on us to figure out the situation.

Krzmbrzl replied:
> it is probably on us to figure out the situation.

Indeed. That's why I'd suggest overtaking the descriptions I provided until someone comes around and complains :D

Krzmbrzl commented on hugo/content/about.md:
```diff
@@ -0,0 +1,73 @@
+
+-->
+
+<!-- Distinction between active and non-active developers? -->
```
I'd rather not do that since this is very prone to become outdated. You could link to https://github.com/mumble-voip/mumble/graphs/contributors though. There the user can filter for the time interval they are interested in and then evaluate for themselves who was active in that period.

toby63 replied:
Well that is not really ideal.
See above :point_up:.

Krzmbrzl commented on hugo/content/about.md:
```diff
@@ -0,0 +1,73 @@
+
+<!-- Distinction between active and non-active developers? -->
+
+<!-- could contain more blabla about vision, free-software dedication etc. -->
```
No real need to do that. We could instead link to a good website explaining what Open Source is about.

toby63 replied:
Well, [wikipedia](https://en.wikipedia.org/wiki/Open-source_model) does not seem to be ideal...
if you know a good website.

Otherwise I will simply leave this out.

Krzmbrzl replied:
I think https://en.wikipedia.org/wiki/Open-source-software_movement is not too bad for this purpose :thinking:

toby63 replied:
What about calling Mumble `Free Software` and link to the [Wikipedia article](https://en.wikipedia.org/wiki/Free_software)?

Krzmbrzl replied:
To be really correct, Mumble is FOSS (Free and Open Source Software). That basically covers everything. Thus https://en.wikipedia.org/wiki/Free_and_open-source_software is probably the most correct thing to link here :thinking:

toby63 replied:
I just saw that you already describe mumble as "Mumble is a free, open source, low latency, high quality voice chat application.", so how should I include it?
For now I have only mentioned it in the section about "Licenses".

Krzmbrzl replied:
You could simply turn "free, open source" into a link to that Wikipedia page :thinking:

Krzmbrzl commented on hugo/content/about.md:
```diff
@@ -0,0 +1,73 @@
+<!-- could contain more blabla about vision, free-software dedication etc. -->
+
+
+<!-- Credits/List of Contributors ? -->
```
Not here. The user may again be referred to https://github.com/mumble-voip/mumble/graphs/contributors

Krzmbrzl commented on hugo/content/about.md:
```diff
@@ -0,0 +1,73 @@
+
+<!-- Credits/List of Contributors ? -->
+
+<!-- https://www.mumble.info/AUTHORS -->
```
I would not link to that since it is probably outdated and furthermore it is exposing their mail addresses. Linking that on a website will just attract web crawlers.
Besides: All contributors can be found in the GitHub stats (or `git log` if someone really wants to know it)

toby63 replied:
Ok, will use the stats/graph then.

Krzmbrzl commented on hugo/content/about.md:
```diff
@@ -0,0 +1,73 @@
+
+
+
+<!-- add category? 
+# Contribute
```
Having this section on the `About` page seems a bit odd anyways :thinking:

toby63 replied:
You are right, we have the general contribute page anyways, so it's visible on first sight.
I was just looking at other websites and also found this.

toby63 commented on hugo/content/contact.md:
```diff
@@ -0,0 +1,38 @@
+
+**For legal matters:**
+
+<!-- add special adress for legal matters? -->
```
Do you already have a special emailadress for this and if not, do you want to create one?

Krzmbrzl replied:
I don't think so. To my knowledge we never needed that and if there really is something to discuss they can simply use the contact mail address...

toby63 replied:
Ok.

Krzmbrzl commented on hugo/content/about.md:
```diff
@@ -0,0 +1,58 @@
+
+## Contributors
+
+We are also supported by contributors from the community.
+You find a [List of Contributors on Github](https://github.com/mumble-voip/mumble/graphs/contributors).
```
```suggestion
Without further contributors from within the community, this project would not be possible.
You  can find a [list of contributors on GitHub](https://github.com/mumble-voip/mumble/graphs/contributors).
```

toby63 replied:
I see your intention, but I think we might reformulate that a bit.
Something like: "Contributions from the community are an important part of the project."

I know that's not perfect either, but thats more the direction I would go, for two reasons:
1. language (sounds rather german, no offense)
2. exaggeration (main contributors are from the team and former team; so community is a nice addition, but not like 50% of the project)

toby63 replied:
By the way: Should I also mention honorable former team members or a special thanks?

Krzmbrzl replied:
> language (sounds rather german, no offense)

I'm actually pretty sure that this is a fine english sentence :shrug: .

But I agree: Your formulation is probably better as it is more correct :+1: 

> By the way: Should I also mention honorable former team members or a special thanks?

Nah, I wouldn't do that. People will find out about those when looking at the contributor page on GitHub.

Krzmbrzl commented on hugo/content/contact.md:
```diff
@@ -0,0 +1,31 @@
+
+security [at] mumble [dot] info
+
+**For License-related topics:**>
```
```suggestion
**For License-related topics:**
```

toby63 replied:
Oh, must have missed that.

toby63 commented on hugo/content/about.md:
```diff
@@ -0,0 +1,58 @@
+
+## Core Team
+
+- Davide Beatrici [davidebeatrici](https://github.com/davidebeatrici) - Core developer
+- Robert Adam [Krzmbrzl](https://github.com/Krzmbrzl)- Core developer, GitHub Management
+- Nik Reist [ZeroAbility](https://github.com/ZeroAbility) - Release Management
+- Jan Klass [Kissaki](https://github.com/Kissaki) - Community work
```
Honestly I think that is a bit too much "on GitHub".
People will directly see where it goes to, so there is no need imo for clarifying that.

Krzmbrzl replied:
They will only see it when hovering over the link. And without doing that having these kind of names simply floating there without context is pretty confusing imo

toby63 replied:
What about a table instead?
With that I could mention Github just once and it would also look nice and clean.

Krzmbrzl replied:
I'd be fine with that as well

toby63 commented on hugo/content/about.md:
```diff
@@ -0,0 +1,58 @@
+
+# About us
+
+We are a team of voluntary developers - both professional and non-professional - from various countries, including Italy, Germany, USA and more.
```
I don't see a reason for that.

Krzmbrzl replied:
Afaik this is the usual way to use this kind of formulation (not 100% sure though) :shrug:

toby63 replied:
Never seen that, a quick search shows that there are many examples of "both professional and non-professional", so I don't think the ":" are necessary.

toby63 commented on hugo/content/about.md:
```diff
@@ -0,0 +1,58 @@
+
+Mumble is [Free and Open-Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software) and licensed under a custom BSD-style [License](https://www.mumble.info/LICENSE).
+
+All Licenses are also listed on GitHub:
+
+- [Mumble Repo (Client & Server)](https://github.com/mumble-voip/mumble/)
+    - [Main Mumble License](https://github.com/mumble-voip/mumble/blob/master/LICENSE)
+
+- [Other Mumble Projects (Overview)](https://github.com/mumble-voip/)
```
Maybe, but I will rewrite it a bit.

I will list the main license and a link to the general github overview and tell people to look at the repos for further licenses.

toby63 commented on hugo/content/about.md:
```diff
@@ -0,0 +1,58 @@
+
+## Licenses
+
+Mumble is [Free and Open-Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software) and licensed under a custom BSD-style [License](https://www.mumble.info/LICENSE).
```
If thats the case, I can say that it is a BSD-license.
I read somewhere that it is costum and only inspired by bsd licenses, but I can't find the source now.

Krzmbrzl replied:
Comparing https://github.com/mumble-voip/mumble/blob/master/LICENSE to https://opensource.org/licenses/BSD-3-Clause shows that in our license file there are a few extra lines mentioning the list of contributors. But as far as I can tell this doesn't affect the legal stuff so in that regards it should be the same as the BSD-3-Clause license

toby63 commented on hugo/config/_default/menus.toml:
```diff
@@ -34,6 +34,12 @@
     identifier = "sponsors"
     url = "/sponsors"
 
+[[main]]
+    title = "About"
+    weight = 7
+    identifier = "about"
+    url = "/about"
```
I would be ok with that, but I think you maybe discuss this in the team once more, because someone has made this sponsors site just recently.

Krzmbrzl replied:
I did check with them and they are fine with that as well :+1:

toby63 commented on hugo/config/_default/menus.toml:
```diff
@@ -45,3 +45,15 @@
+    title = "Sponsors"
+    weight = 4
+    identifier = "sponsors"
+    url = "/sponsors"
```
Correct position?

Krzmbrzl replied:
Are you referring to the entry's position in the footer? I don't mind where that is listed :shrug:

toby63 replied:
Yes. Ok, I guess we will see what others have to say on review.

toby63 commented on hugo/content/about.md:
```diff
@@ -0,0 +1,66 @@
+| Robert Adam | [Krzmbrzl](https://github.com/Krzmbrzl) | Core developer, GitHub Management |
+| Nik Reist | [ZeroAbility](https://github.com/ZeroAbility) | Release Management |
+| Jan Klass | [Kissaki](https://github.com/Kissaki) | Community work |
+| Natenom | [Natenom](https://github.com/Natenom) | Testing |
```
Example table.

Krzmbrzl replied:
In case of Natenom I would leave the real name entry blank.

toby63 replied:
:thinking: I think leaving an empty space is not so good...
How about clarifying that it is a nickname?
Like `Natenom (Nickname)`

Krzmbrzl replied:
If you don't want to leave it blank, fill the table cell with a single `-` to symbolize that the real name is not provided.

toby63 replied:
Well, if you want it like that.

toby63 commented on hugo/content/contact.md:
```diff
@@ -0,0 +1,31 @@
+
+**General contact:**
+
+contact [at] mumble [dot] info
```
Optional: Do you have an encryption key?

Krzmbrzl replied:
For email? I don't think so

toby63 replied:
Yes. Ok.

toby63 commented:
So I have moved **Contact** to the first place in the footer.
And added **About** to the main menu and moved **Sponsors** to the footer.

Kissaki commented on hugo/content/contact.md:
```diff
@@ -0,0 +1,31 @@
+
+## Community Channels
+
+* Our [issue tracker on GitHub](https://github.com/mumble-voip/mumble/issues)
```
Prefixing/Introing with _Our_ here is inconsistent with the other bullet points.

```suggestion
* [Issue tracker on GitHub](https://github.com/mumble-voip/mumble/issues)
```

toby63 replied:
I wanted to clarify with that, that it belongs to mumble.
Maybe I would reformulate it, like:
`[Issue Tracker in Mumble's Github Repo]`

Krzmbrzl replied:
I think it should be clear from the context that this is an issue tracker related to Mumble :thinking:

toby63 replied:
> I think it should be clear from the context that this is an issue tracker related to Mumble :thinking:

That might be correct.

toby63 replied:
Ok, I will just name it `Issue tracker on GitHub`.

Kissaki commented on hugo/content/contact.md:
```diff
@@ -0,0 +1,31 @@
+* [Discussions on GitHub](https://github.com/mumble-voip/mumble/discussions)
+* IRC: [#mumble on irc.freenode.net](irc://irc.freenode.org/mumble)
+* Chat: [Mumble room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
+* Our [Forums](https://forums.mumble.info/)
```
```suggestion
* [Forums](https://forums.mumble.info/)
```

toby63 replied:
1. Should we use `Forum` instead?
2. Should I add `Official Forum`?

Kissaki replied:
All of these are official resources. I think just Forum or Forums is fine.

Kissaki commented on hugo/content/contact.md:
```diff
@@ -0,0 +1,31 @@
+
+* Our [issue tracker on GitHub](https://github.com/mumble-voip/mumble/issues)
+* [Discussions on GitHub](https://github.com/mumble-voip/mumble/discussions)
+* IRC: [#mumble on irc.freenode.net](irc://irc.freenode.org/mumble)
+* Chat: [Mumble room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
```
I think this should also be made more consistent. Chat is very unspecific, and IRC is also a chat.

With the other bullet points being mostly full links it seems good to make them here so as well.

```suggestion
* [IRC #mumble on freenode](irc://irc.freenode.org/mumble)
* [Mumble chat room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
```

toby63 replied:
You are correct with this, but I would change it differently:
```

* Chat:
    * IRC: bla
    * Matrix: bla
```

Kissaki commented on hugo/content/contact.md:
```diff
@@ -0,0 +1,31 @@
+title: Contact
+toc: true
+---
+
+# Contact
+
```
This adds a duplicate headline to the rendered page. Did you miss that on testing?

```suggestion

```

toby63 replied:
Seems that way. Ok, will use the title tag only instead.

Kissaki commented on hugo/content/contact.md:
```diff
@@ -0,0 +1,31 @@
+---
+title: Contact
+toc: true
```
This seems to enable a table of content.

But I do not see any toc being rendered!?

toby63 replied:
Well I haven't checked this in the docs, but with other software I use (for some other projects) this tag can have different meanings, for example it means to enable an indexing of the website (so that you have section links to each category on the page).

Kissaki replied:
It’s a parameter that we currently do not use. If we were to enable it here, it could lead to unforeseen consequences once we add toc handling to the layout templates. We don’t know what would happen.

It should be removed to prevent unknown risks while not adding or doing anything.

toby63 replied:
Ok, just for clarification:
1. Without this option we would still have section links like: https://www.mumble.info/contribute/#report-bugs-and-feature-requests ?
2. Is this a custom parameter?
3. This parameter is used on other pages too, should we remove it everywhere?

Kissaki replied:
Yes, [contribute.md](https://github.com/mumble-voip/mumble-www/blob/master/hugo/content/contribute.md) manually creates a list with links as content, and depends on the headline text which leads to an id attribute to match the link that is declared.

If toc were implemented like this, we would suddenly have two table of content lists.

If a headline changes, the manual list has to be updated too atm.

When I checked our layout did not have toc handling. So yes, it should IMO be removed from other pages too.

Given that the params docs gives an example for `include_toc` as a parameter, I think `toc` as it was used here is a user defined parameter too. There seems to be no other doc on `toc`, which supports that theory.

Kissaki commented on hugo/content/about.md:
```diff
@@ -0,0 +1,58 @@
+title: About
+toc: true
+---
+
+# About the project
```
Double headline

![image](https://user-images.githubusercontent.com/93181/107993305-964b4100-6fda-11eb-919d-551041190a51.png)

```suggestion
```

toby63 replied:
Ok, will change.

Kissaki commented on hugo/content/about.md:
```diff
@@ -0,0 +1,58 @@
+---
+title: About
+toc: true
```
Declares that it enables a toc, but I do not see any when rendering the page??

```suggestion
```

Kissaki commented on hugo/content/about.md:
```diff
@@ -0,0 +1,58 @@
+
+## Licenses
+
+Mumble is [Free and Open-Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software) and licensed under a BSD-style [License](https://www.mumble.info/LICENSE).
```
Suggestion: We already link FOSS above. I would drop it here.

IMO linking the word _License_ here does not read fluent. It would be better, but still not optimal, to link _a BSD-style License_. I would add another sentence to signify that we are linking to the LICENSE/LICENSE file specifically here.

```suggestion
Mumble is Free and Open-Source Software and licensed under [a BSD-style License](https://www.mumble.info/LICENSE).
```

```suggestion
Mumble is Free and Open-Source Software and licensed under a BSD-style License. See our [LICENSE file](https://www.mumble.info/LICENSE) for details.
```

Side note: We should probably mention our varying used licenses here, not just the project source code, and provide a reasonable intro to that. But this is fine for a starter.

toby63 replied:
> IMO linking the word License here does not read fluent. It would be better, but still not optimal, to link a BSD-style License. I would add another sentence to signify that we are linking to the LICENSE/LICENSE file specifically here.

I agree, so I will use your second suggestion, correct?

> Side note: We should probably mention our varying used licenses here, not just the project source code, and provide a reasonable intro to that. But this is fine for a starter.

It is mentioned below, but it can be improved I guess.

Kissaki replied:
Yes

Kissaki commented on hugo/content/about.md:
```diff
@@ -0,0 +1,58 @@
+
+## Contact
+
+Take a look at our [Contact page](https://www.mumble.info/contact).
```
This should be a relative hugo native link.

toby63 replied:
I don't like it, but I will add it anyway.

Kissaki replied:
Do you have a reason for disliking it?

Not using native, relative links leads to technical issues if you make changes to how and where you deploy.

toby63 replied:
I like and understand the concept of relative linking; what I dislike is described here: #135.
In the meantime I already added relref links in my offline version, which I will upload along with all other changes, once it's ready.

Kissaki commented on hugo/content/about.md:
```diff
@@ -0,0 +1,58 @@
+
+# Security notes
+
+Our [Security page](https://www.mumble.info/security) contains information on security issues, support & contact.
```
Should be a relative, hugo native link.

Kissaki commented on hugo/content/about.md:
```diff
@@ -0,0 +1,58 @@
+
+# Privacy notes
+
+Our Privacy policy can be found [here](https://www.mumble.info/privacy).
```
Should be a relative, hugo native link.

Kissaki reviewed (changes requested):
Suggestion: It would be nice if the commit description and PR description would mention the menu being moved, and reasoning for it. (Space)
That’s a significant change of existing concept/content.

Thought for later: I think we have to implement clickable and readable links and should to so eventually. I presume you put it in like this so they are not easily discoverable by bots/crawlers. Which is fine for now.

Kissaki commented:
Sorry for the delay in me reviewing, and the many comments. I hope you do not take change comments personal, and I hope it does not frustrate you too much or make you stop with this. I know reviews can be exhausting. :)

Krzmbrzl commented on hugo/content/about.md:
```diff
@@ -0,0 +1,58 @@
+| Davide Beatrici | [davidebeatrici](https://github.com/davidebeatrici) | Core developer |
+| Robert Adam | [Krzmbrzl](https://github.com/Krzmbrzl) | Core developer, GitHub Management |
+| Nik Reist | [ZeroAbility](https://github.com/ZeroAbility) | Release Management |
+| Jan Klass | [Kissaki](https://github.com/Kissaki) | Community work |
```
I feel this should represents the current situation (that's also why Thorvald isn't listed at all).
Nonetheless you are free to suggest a different role description here. This was just my original suggestion and since you did not answer on my ping on this topic, we left it at that :shrug:

toby63 replied:
@Krzmbrzl Could you maybe talk about this in one of your team meetings, so that we could have an accurate representation of every team member?
@Kissaki What would you like as a description?

Kissaki replied:
Whatever

toby63 replied:
"Whatever" is not really a good answer. Maybe other team members want a different description etc. as well. I will wait for the results of your team meeting, thus will unresolve this.

toby63 commented on hugo/content/about.md:
```diff
@@ -0,0 +1,58 @@
+
+Active Team Members are:
+
+| Name: | Github Profile: | Assignment: |
```
I think colons look better, but if you don't want to have them.

toby63 commented on hugo/config/_default/menus.toml:
```diff
@@ -29,19 +29,32 @@
     weight = 1
+    identifier = "contact"
+    url = "/contact/"
+    
```
I remember that I checked this, but I can check again.

toby63 replied:
You are correct, will change it.

toby63 commented on hugo/content/contact.md:
```diff
@@ -0,0 +1,31 @@
+
+# Contact
+
+You can contact us in several ways:
```
I think that is too long text, but if you want it this way.

toby63 commented:
> Sorry for the delay in me reviewing, and the many comments. I hope you do not take change comments personal, and I hope it does not frustrate you too much or make you stop with this. I know reviews can be exhausting. :)

That is not a problem for me.
But the changes will probably take some time.

toby63 commented:
> Suggestion: It would be nice if the commit description and PR description would mention the menu being moved, and reasoning for it. (Space)
That’s a significant change of existing concept/content.

I will add it.

> Thought for later: I think we have an opportunity to implement clickable and readable links and should do so eventually. I presume you put it in like this so they are not easily discoverable by bots/crawlers. Which is fine for now.

I don't really know what you mean, do you refer to the emailadresses?

Kissaki commented:
> > Thought for later: I think we have an opportunity to implement clickable and readable links and should do so eventually. I presume you put it in like this so they are not easily discoverable by bots/crawlers. Which is fine for now.
> 
> I don't really know what you mean, do you refer to the emailadresses?

Yes, sorry. That was referring to the email addresses.

Krzmbrzl commented on hugo/content/about.md:
```diff
@@ -0,0 +1,57 @@
+
+## Source Code & Development
+
+The Source Code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).
```
```suggestion
The source code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).
```

Krzmbrzl commented on hugo/content/about.md:
```diff
@@ -0,0 +1,57 @@
+
+The Source Code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).
+
+For other Mumble Projects, see [Overview of Repositories on GitHub](https://github.com/mumble-voip/).
```
```suggestion
For other Mumble Projects, see the [overview of repositories on GitHub](https://github.com/mumble-voip/).
```

Krzmbrzl commented on hugo/content/about.md:
```diff
@@ -0,0 +1,57 @@
+## Licenses
+
+Mumble is Free and Open-Source Software.
+The core project is licensed under a BSD-style License. See our [LICENSE file](https://www.mumble.info/LICENSE) for details.
```
```suggestion
The core project is licensed under a BSD-style license. See our [LICENSE file](https://www.mumble.info/LICENSE) for details.
```

Krzmbrzl commented on hugo/content/about.md:
```diff
@@ -0,0 +1,57 @@
+Mumble is Free and Open-Source Software.
+The core project is licensed under a BSD-style License. See our [LICENSE file](https://www.mumble.info/LICENSE) for details.
+
+There are additional licenses, you can find them in the GitHub repositories, see: [Overview of all Mumble Projects](https://github.com/mumble-voip/)
```
```suggestion
Our other projects may use a different license. For details we refer to the respective [GitHub repositories](https://github.com/mumble-voip/)).
```

Krzmbrzl commented on hugo/content/about.md:
```diff
@@ -0,0 +1,57 @@
+
+Active Team Members are:
+
+| Name | Github Profile | Assignment |
```
```suggestion
| Name | GitHub Profile | Assignment |
```

Krzmbrzl commented on hugo/content/contact.md:
```diff
@@ -0,0 +1,29 @@
+* [Discussions on GitHub](https://github.com/mumble-voip/mumble/discussions)
+* Chat:
+    * IRC: [#mumble on irc.freenode.net](irc://irc.freenode.org/mumble)
+    * Matrix: Join `#freenode_#mumble:matrix.org`; for example via [Riot.im (Web App)](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
```
```suggestion
    * Matrix: Join `#freenode_#mumble:matrix.org`; for example via [Element (Web App)](https://app.element.io/#/room/#mumble:matrix.org) (shares messages with IRC)
```

Kissaki commented on hugo/content/about.md:
```diff
@@ -0,0 +1,53 @@
+---
+title: About
+
```
Remove trailing empty line in front matter
```suggestion
```

Kissaki commented on hugo/content/about.md:
```diff
@@ -0,0 +1,53 @@
+
+The source code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).
+
+For other Mumble Projects, see the [overview of repositories on GitHub](https://github.com/mumble-voip/).
```
GitHub itself links to org without slash, so we should use that. Strangely enough slash does not redirect to without.

```suggestion
For other Mumble Projects, see the [overview of repositories on GitHub](https://github.com/mumble-voip).
```

Kissaki commented on hugo/content/about.md:
```diff
@@ -0,0 +1,53 @@
+Mumble is Free and Open-Source Software.
+The core project is licensed under a BSD-style license. See our [LICENSE file](https://www.mumble.info/LICENSE) for details.
+
+Our other projects may use different licenses. For details we refer to the respective GitHub repositories, see: [Overview of all Mumble projects on GitHub](https://github.com/mumble-voip/).
```
Is this a typo or intended this way?

```suggestion
Our other projects may use different licenses. For details please refer to the respective GitHub repositories, see: [Overview of all Mumble projects on GitHub](https://github.com/mumble-voip/).
```

Krzmbrzl replied:
It was intended this way, but I think your suggestions is better

toby63 replied:
Agreed.

Kissaki commented on hugo/content/contact.md:
```diff
@@ -0,0 +1,29 @@
+---
+title: Contact
+
```
Remove trailing empty line in front matter

```suggestion
```

Kissaki reviewed (approved):
Ok, nice.

I mark some small cleanups. I do not consider them blockers though, but would like them resolved still.

For the commit  message:
Please use present tense for commit messages. "Add pages" instead of "Added pages".
Do not end commit title with a fullstop (it’s not a sentence but a title).

I tried to find a definite reference for this, but have difficulty finding it right now. Anyway, present tense is the convention, starting from the Git project itself.

Otherwise LGTM.

toby63 commented:
Ok, will upload a fix for that.

Some last questions:
1. I noticed that the element.io link (https://app.element.io/#/room/#mumble:matrix.org) requires a registration, so should we really mention that? (it's on the contact page and also on the contribute page)
2. Do you want to mention the sponsors page on the about page?
3. Should we add a special thanks for @thorvald etc.?

Krzmbrzl commented:
1. Yes we absolutely should. Plain IRC is just a nightmare to use if you're not online 24/7. With Element you get persistent messages which is a very important feature that allows you to drop a question and come back later to see if someone answered.
2. I think we don't have to put it there. That's probably not interesting for folks visiting that page
3. I don't think this is interesting for the reader either. If they want to know who contributed, they can visit the Contribute GitHub page and there's a full list. Imo that's enough :shrug:

toby63 commented:
I added the suggestions by @Kissaki.

I also changed the headers.

---

Regarding the questions:
1. One alternative would be that I at least put a note there that it requires registration.
2. I only brought it up, because now the Sponsors page is moved to the footer. So it would get a bit more attention this way.
3. I think some people would deserve it. I also saw it on other projects websites, something like "honorable members" or similar.

Kissaki commented:
The commit is now present tense instead of past, but still does not follow the recommended format.

>  Adds pages "About" & "Contact"
> 
> Adds "About" to the menu and moves "Sponsors" to footer.
> Adds "Contact" to footer.

To follow convention it should be

>  Add pages "About" & "Contact"
> 
> Add "About" to the menu and moves "Sponsors" to footer.
> Add "Contact" to footer.

Think of it not as describing what the commit does, but describing what the change itself does. The change is the subject. The change that is applied to source code. And it is a description of what that change does.

_I feel like this explanation sucks. Anyway, I don’t have anything better right now._

Krzmbrzl commented:
> One alternative would be that I at least put a note there that it requires registration.

Nah I don't like that. People will find that out simply by clicking the link. And it is not like they are automatically registering anywhere just by clicking the link.
Besides: I think that 99% of all users visiting that link also _don't_ expect to be able to participate without registration :shrug: 

> I think some people would deserve it. I also saw it on other projects websites, something like "honorable members" or similar.

_Personally_ I am not interested in this kind of stuff when I look at such a page. If you (or someone else) think that it should be done though, I don't have any objections to adding such a section to the bottom.
Doing so would however raise the question of when a contributor will be listed here :shrug:

toby63 commented:
@Kissaki I understand, though I think it's a little weird, but you are correct that it is mostly done this way.

@Krzmbrzl Ok, in that case I think we should use this as is and maybe someone will add something in the future.

Ready for merge.

Krzmbrzl commented:
Thank you for your contribution :)
